src/Apps/Collect/Routes/Collection/Components/Header/__tests__/fixtures/artworks.ts

```diff
@@ -64,14 +65,13 @@ export const collectionHeaderArtworks: Header_artworks = {
   ],
 }
 
-export const defaultCollectionHeaderArtworks: Header_artworks &
-  DefaultHeader_headerArtworks = {
-  " $refType": null as never,
-  " $fragmentRefs": null as never,
-  merchandisableArtists: null,
+export const defaultCollectionHeaderArtworks: {
+  edges: Array<DeFraged<DefaultHeaderArtwork_artwork>>
```

Comment:
This `DeFraged` should be a DeepOmit. We kinda should just install ts-essentials me-thinks.